monitoring/uss_qualifier/run_locally_rid.sh

```diff
@@ -24,18 +24,37 @@ monitoring/build.sh || exit 1
 CONFIG_LOCATION="monitoring/uss_qualifier/config_run_locally_rid.json"
 CONFIG='--config config_run_locally_rid.json'
 
-AUTH='--auth DummyOAuth(http://host.docker.internal:8085/token,uss_qualifier)'
+AUTH_SPEC='DummyOAuth(http://host.docker.internal:8085/token,uss_qualifier)'
+AUTH_FLAG="--auth ${AUTH_SPEC}"
 
 echo '{
     "locale": "CHE",
+    "resources": {
+      "resource_declarations": {
+        "utm_auth": {
+          "resource_type": "communications.AuthAdapter",
+          "specification": {
+            "environment_variable_containing_auth_spec": "AUTH_SPEC"
```

Comment:
Do we need to keep it from the environment? Perhaps it could simply be included directly.

Reply:
Yes, since this is a test configuration with no sensitive information, we could put it directly in the config file and there's an option to do that in the specification.  But, my thought is that most real-world applications will have sensitive data in the auth spec and people generally won't want to put that data directly in the configuration (which is sometimes published fairly broadly), so I figured I'd set an example here.  That's the same logic that lead the auth spec to be passed via a flag in the initial implementation rather than being put directly in the configuration.